cmake/libappimageConfig.cmake.in

```diff
@@ -3,23 +3,26 @@
 #  LIBAPPIMAGE_INCLUDE_DIRS - include directories for LIBAPPIMAGE
 #  LIBAPPIMAGE_LIBRARIES    - libraries to link against
 
+@PACKAGE_INIT@
+
 # Compute paths
 get_filename_component(LIBAPPIMAGE_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-set(LIBAPPIMAGE_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
 
 # Our library dependencies (contains definitions for IMPORTED targets)
 include("${LIBAPPIMAGE_CMAKE_DIR}/libappimageTargets.cmake")
 
+# TODO: while we check here for the libs with pkgconfig,
+# we do not create here the respective targets as used
+# in the link interface of the exported libappimage* targets
+# Those target names need to be either mapped here to those
+# pkg_check_modules(IMPORTED_TARGET) creates or changed
+# in the libappimage buildsystem to the original ones.
```

Comment:
You mean like here https://github.com/AppImage/libappimage/blob/ae0258ab484c1259facc8fd85aaa8c7857c3e155/src/CMakeLists.txt#L6 ?

Reply:
I guess it's an anti pattern to do that.